gas-estimation/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "gas-estimation"
+version = "0.1.0"
+authors = ["Gnosis Protocol Team <dex@gnosis.io>"]
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+log = "0.4"
+serde = { version = "1.0", features = ["derive"] }
+serde_with = "1.5"
+web3 = "0.13"
+primitive-types = { version = "0.7.3", features = ["fp-conversion"] }
```

Comment:
Have you already made the appropriate changes related to this updated dependency? I feel like this should be a separate PR, but could be left here if it doesn't include all the old conversion call site changes as well.

Reply:
I have not updated the dependency in other parts (or replaced the num_to_f64 implementation). I think this is its own PR, but since I'm introducing new code here I  though I might as well use the new version instead of copy and pasting deprecated code again.